packages/swing-store/package.json

```diff
@@ -21,13 +21,13 @@
     "@agoric/assert": "^0.5.1",
     "@agoric/internal": "^0.2.1",
     "better-sqlite3": "^7.5.0",
-    "lmdb": "^2.4.5",
     "tmp": "^0.2.1"
   },
   "devDependencies": {
     "@endo/init": "^0.5.52",
     "@types/better-sqlite3": "^7.5.0",
     "ava": "^5.0.1",
+    "@agoric/swingset-vat": "^0.30.2",
```

Comment:
Yes, but it's Ava and not `assert` that's the culprit.  This could definitely be handled better, as it's sort of a path-dependent accident of history that some of our test infra lives in the `SwingSet` package.  However, I don't think this is the PR to address that, as there seem to be 10-20 other `agoric-sdk` packages that have the same problem.  An open question is where this test boilerplate should live instead, for which I don't have an immediate obvious answer.  Somebody should file an issue...

Reply:
Filed #6703 about making `prepare-test-env-ava.js` into a separate package somehow.